pkg/webhooks/validator/validator_test.go

```diff
@@ -1220,6 +1223,43 @@ var _ = Describe("webhooks validator", func() {
 				Entry("should not trigger warning if enableCommonBootImageImport (true) wasn't changed", ptr.To(true), ptr.To(true)),
 				Entry("should not trigger warning if enableCommonBootImageImport (false) wasn't changed", ptr.To(false), ptr.To(false)),
 			)
+
+			//nolint:staticcheck
+			DescribeTable("should return warning for deployVmConsoleProxy on update", func(newFG, oldFG *bool) {
+				newHCO := hco.DeepCopy()
+				hco.Spec.FeatureGates.DeployVMConsoleProxy = newFG
+				newHCO.Spec.FeatureGates.DeployVMConsoleProxy = oldFG
+
+				err := wh.ValidateUpdate(ctx, dryRun, newHCO, hco)
+
+				Expect(err).To(HaveOccurred())
+				expected := &ValidationWarning{}
+				Expect(errors.As(err, &expected)).To(BeTrue())
+
+				Expect(expected.warnings).To(HaveLen(1))
+				Expect(expected.warnings).To(ContainElements(ContainSubstring("deployVmConsoleProxy")))
+			},
+				Entry("should trigger warning if deployVmConsoleProxy appeared as true", nil, ptr.To(true)),
+				Entry("should trigger warning if deployVmConsoleProxy appeared as false", nil, ptr.To(false)),
+				Entry("should trigger warning if deployVmConsoleProxy has changed from true to false", ptr.To(true), ptr.To(false)),
+				Entry("should trigger warning if deployVmConsoleProxy has changed from false to true", ptr.To(false), ptr.To(true)),
+			)
+
+			//nolint:staticcheck
```

Comment:
I would add a space at the start of the line like other places in the pr

Reply:
same